src/Compilers/VisualBasic/Portable/Symbols/ReferenceManager.vb

```diff
@@ -604,48 +596,51 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Sub
 
             Private Shared Sub SetupReferencesForSourceAssembly(
+                allAssemblyData As ImmutableArray(Of AssemblyData),
```

Comment:
I would expect to see a change around this parameter similar to the change made on C# side.